buku

```diff
@@ -36,9 +36,10 @@ import threading
 import time
 from typing import Any, Dict, List, Optional, Tuple, Union
 import webbrowser
+import pyreadline
 try:
-    import readline
```

Comment:
i would recommend this method

https://stackoverflow.com/a/6024996/1766261

that way, other who can still use readline will still use it

Reply:
That's good, I like that. I don't develop with python a great deal unless I have to but I guess I could have found that solution on my own eventually. Thanks for sharing. :)